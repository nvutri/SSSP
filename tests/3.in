c random graph n=32 m=128 lo=0 hi=64 seed=1913762381
p sp 32 128
a 14 11 11
a 11 28 21
a 28 23 42
a 23 15 21
a 15 19 47
a 19 30 48
a 30 21 47
a 21 6 27
a 6 4 12
a 4 5 31
a 5 31 22
a 31 3 26
a 3 8 12
a 8 32 1
a 32 17 40
a 17 12 12
a 12 22 60
a 22 9 29
a 9 20 42
a 20 26 8
a 26 1 56
a 1 18 38
a 18 7 4
a 7 27 42
a 27 2 14
a 2 13 28
a 13 24 9
a 24 25 43
a 25 10 14
a 10 16 29
a 16 29 41
a 29 14 58
a 15 17 46
a 10 7 52
a 9 11 27
a 24 28 24
a 26 9 10
a 20 13 33
a 27 3 32
a 24 28 58
a 24 21 60
a 25 9 47
a 31 8 51
a 8 5 5
a 26 16 7
a 9 7 44
a 5 18 11
a 6 26 48
a 2 12 5
a 3 10 50
a 5 7 11
a 22 32 38
a 31 7 34
a 24 19 16
a 24 3 55
a 15 22 45
a 21 14 53
a 26 25 23
a 13 23 32
a 20 12 41
a 32 28 3
a 31 21 14
a 24 4 23
a 13 25 60
a 8 6 11
a 29 13 5
a 8 32 30
a 20 8 3
a 7 16 34
a 19 27 10
a 7 6 40
a 16 4 38
a 11 24 27
a 13 32 32
a 7 31 48
a 19 27 6
a 9 7 2
a 23 28 59
a 29 26 1
a 29 11 46
a 27 8 26
a 7 4 22
a 21 6 52
a 13 31 34
a 28 20 24
a 22 15 47
a 28 7 32
a 5 11 11
a 2 27 53
a 15 13 19
a 9 22 33
a 7 24 49
a 7 11 18
a 16 31 17
a 28 13 41
a 16 5 6
a 10 5 1
a 7 28 39
a 6 7 44
a 18 13 31
a 29 23 19
a 14 23 21
a 13 26 53
a 30 32 42
a 2 27 43
a 15 28 9
a 18 12 30
a 21 15 39
a 12 15 33
a 3 21 56
a 5 11 12
a 8 22 54
a 10 26 13
a 17 10 42
a 25 11 29
a 2 16 19
a 31 6 30
a 20 25 44
a 6 12 16
a 5 19 9
a 11 16 55
a 3 27 4
a 13 8 44
a 12 28 47
a 2 23 22
a 3 18 5
a 6 22 27
a 20 24 17
