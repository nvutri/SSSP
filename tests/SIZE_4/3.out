1 0
2 98
3 116
4 64
5 82
6 82
7 42
8 108
9 123
10 135
11 60
12 68
13 69
14 157
15 101
16 76
17 141
18 38
19 91
20 105
21 104
22 109
23 101
24 78
25 121
26 113
27 84
28 81
29 117
30 139
31 90
32 101
